src/pages/Troubleshooting.vue

```diff
@@ -57,7 +57,7 @@
             data-test="factoryResetButton"
             label="Factory Reset"
             value="auto"
-            :options="[{id: 'keepImages', label: 'Keep Container System Images'}]"
+            :options="[{id: 'keepImages', label: 'Keep cached kubernetes images'}]"
```

Comment:
`Kubernetes` should be capitalized, but I'm fine not capitalizing the other two words.